package.json

```
@@ -1,187 +1,22 @@
-  "private": false,
-  "license": "SEE LICENSE IN LICENSE.md",
-  "author": "Shopify <dev@shopify.com>",
-  "homepage": "https://polaris.shopify.com/components",
-  "repository": "https://github.com/Shopify/polaris-react",
-  "bugs": {
-    "url": "https://github.com/Shopify/polaris-react/issues"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "files": [
-    "build/",
-    "!build/ts/**/*.tsbuildinfo",
-    "!build/ts/**/tests/",
-    "!build/ts/**/playground/",
-    "locales/"
-  ],
-  "sideEffects": [
-    "**/*.css",
-    "**/*.scss",
-    "**/configure.{js,mjs,esnext,ts}"
-  ],
-  "keywords": [
-    "shopify",
-    "polaris",
-    "react",
-    "components",
-    "component library"
-  ],
-  "main": "build/cjs/index.js",
-  "module": "build/esm/index.js",
-  "esnext": "build/esnext/index.esnext",
-  "types": "build/ts/latest/src/index.d.ts",
-  "typesVersions": {
-    "<4.0": {
-      "build/types/latest/*": [
-        "build/ts/3.4/*"
-      ]
-    }
-  },
+  "name": "polaris-monorepo",
+  "private": true,
   "engines": {
+    "npm": ">=7.0.0",
     "node": ">=16.13.0"
   },
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "lint": "loom lint",
-    "format": "loom lint --fix",
-    "type-check": "loom type-check",
-    "a11y-check": "node ./scripts/accessibility-check.js",
-    "test": "loom test",
-    "check": "npm-run-all lint type-check test",
-    "clean": "rimraf build build-internal",
-    "build": "node ./scripts/build.js",
-    "prebuild-consumer": "yarn run build",
-    "build-consumer": "node ./scripts/build-consumer",
-    "secrets": "ejson decrypt -o secrets.json secrets.ejson",
-    "prenew-version-pr-generator": "rimraf sandbox && yarn run secrets",
-    "new-version-pr-generator": "node ./scripts/new-version-pr-generator",
-    "prepublishOnly": "yarn run build",
-    "dev": "yarn run storybook",
-    "start": "serve ./build-internal/storybook/static -l ${PORT:=6006}",
-    "readme-update-version": "node ./scripts/readme-update-version",
-    "version": "yarn run readme-update-version",
-    "storybook": "start-storybook -p 6006 --quiet",
-    "storybook:build": "build-storybook -o build-internal/storybook/static",
-    "validate-tokens": "ajv validate -s src/tokens/schemas/token-group.json -d 'src/tokens/token-groups/*.json'"
-  },
-  "dependencies": {
-    "@shopify/polaris-icons": "^4.17.0",
-    "@types/react": "^17.0.19",
-    "@types/react-dom": "^17.0.9",
-    "@types/react-transition-group": "^4.4.2",
-    "focus-visible": "^5.2.0",
-    "lodash": "^4.17.4",
-    "react-transition-group": "^4.4.2"
-  },
-  "peerDependencies": {
-    "react": "^16.14.0 || ^17.0.0",
-    "react-dom": "^16.14.0 || ^17.0.0"
+    "build": "yarn workspaces run build",
+    "dev:components": "yarn --cwd packages/components dev",
+    "dev:website": "yarn --cwd packages/website dev",
+    "build:components": "yarn --cwd packages/components build",
+    "build:website": "yarn --cwd packages/website build"
```

```suggestion
    "dev:components": "yarn workspace components dev",
    "dev:website": "yarn workspace website dev",
    "build:components": "yarn workspace components build",
    "build:website": "yarn workspace website build"
```

**question:** Should we be using the `workspace` subcommand?

---

Resolved